### ISSUE TITLE
Issues: Tables are not rendered correctly

### ISSUE THREAD

**Issue by lamyergeier** [0x6b/copy-selection-as-markdown#98]
Source: [Debits and credits - Wikipedia](https://en.m.wikipedia.org/wiki/Debits_and_credits), under the heading: - [Aspects of transactions](https://en.m.wikipedia.org/wiki/Debits_and_credits#Aspects_of_transactions)

**Actual Result**
On the other hand, increases in revenue, liability or equity accounts are credits or right side entries, and decreases are left side entries or debits.

 Kind of account

Debit

Credit

**Asset**

**Increase**

Decrease

Liability

Decrease

**Increase**

Income/Revenue

Decrease

**Increase**

**Expense/Cost/Dividend**

**Increase**

Decrease

Equity/Capital

Decrease

**Increase**

Debits and credits occur simultaneously in every [financial](https://en.m.wikipedia.org/wiki/Financial_accounting "Financial accounting") transaction in double-entry bookkeeping.

 

**Expected Result** (Using [clipboard2markdown](http://euangoddard.github.io/clipboard2markdown/))
On the other hand, increases in revenue, liability or equity accounts are credits or right side entries, and decreases are left side entries or debits.

|  Kind of account | Debit | Credit |
---|---|---|
| **Asset** | **Increase** | Decrease |
| Liability | Decrease | **Increase** |
| Income/Revenue | Decrease | **Increase** |
| **Expense/Cost/Dividend** | **Increase** | Decrease |
| Equity/Capital | Decrease | **Increase** |

Debits and credits occur simultaneously in every [financial](https://en.m.wikipedia.org/wiki/Financial_accounting "Financial accounting") transaction in double-entry bookkeeping.

**Comment by 0x6b** [comment]
Thanks for the issue @anishmittal2020. The part works for me. Can you please check your addon version? It should be:

- Version 0.21.0
- Last Updated June 27, 2021

Update to the latest by `about:addons` → **Gear** icon at top-right → **Check for updates**, restart your browser, and retry please. As it's works for me it might be difficult to investigate and/or fix I'm afraid.

**Comment by anishmittal2020** [comment]
@0x6b I have same Addon version, and I tried once again, I got exeactly the same issue.

Firefox: 90
OS: Ubuntu 20.04
Extension Version: 0.21.0
Last Updated: 26 June 2021

**Comment by 0x6b** [comment]
@anishmittal2020 Can you try the page with checking **Enable GitHub Flavored Markdown (strikethrough, tables, and taskListItems) support**? [Original Markdown syntax](https://daringfireball.net/projects/markdown/syntax) does not support table so you have to check the preference.

**Comment by anishmittal2020** [comment]
Hi what you mentioned worked for the aforemnetioned example. I changed my settings:


![image](https://user-images.githubusercontent.com/42092626/135589561-6deed4c5-98db-48dd-9db7-f4fc59ae991b.png)

But it does not work in this example: [Attitudes, Beliefs, and Values  SPCH 1311: Introduction to Speech Communication](https://courses.lumenlearning.com/atdcoursereview-speechcomm-1/chapter/attitudes-beliefs-and-values/)

|           | Definition                                     | Changeable?            | Example                                                                                         |
| --------- | ---------------------------------------------- | ---------------------- | ----------------------------------------------------------------------------------------------- |
| Attitudes | Learned predispositions to a concept or object | Subject to change      | I enjoyed the writing exercise in class today.                                                  |
| Beliefs   | Convictions or expressions of confidence       | Can change over time   | This course is important because I may use the communication skills I am learning in my career. |
| Values    | Ideals that guide our behavior                 | Generally long lasting | Effective communication is important.                                                           |

It weorks with [clipboard2markdown](http://euangoddard.github.io/clipboard2markdown/))

**Comment by 0x6b** [comment]
@lamyergeier Thank you for your patience. After nearly 3 years of silence, I've released version 0.22.0, and it seems to be working fine with the URL. Please check and if you have something strange, please reopen the issue. Thanks!

**Comment by lamyergeier** [comment]
@0x6b  The following table from [File attribute - Wikipedia](https://en.wikipedia.org/wiki/File_attribute) is rendered as below

![image](https://github.com/0x6b/copy-selection-as-markdown/assets/42092626/18826317-ec4e-4773-be3d-48dabea44aa5)


```
Common file attributes supported by many common Linux file systems Attribute .mw-parser-output .monospaced{font-family:monospace,monospace}lsattr flag chattr option Semantics and rationale No `atime` updates `A` `+A,-A` [[atime record](https://en.wikipedia.org/wiki/Stat_(Unix))](https://en.wikipedia.org/wiki/Stat_(Unix) "Stat (Unix)") is not modified when file is read/accessed. Append-only `a` `+a,-a`

Writing to file only allowed in append mode.

Immutable `i` `+i,-i`

Prevents any change to file's contents or metadata: file/directory cannot be written to, deleted, renamed, or hard-linked.

No dump `d` `+d,-d` File is skipped by the [[dump program](https://en.wikipedia.org/wiki/Dump_(Unix))](https://en.wikipedia.org/wiki/Dump_(Unix) "Dump (Unix)") Secure deletion `s` `+s,-s` Requests that, when deleted, all file data blocks are [[filled with zeroes](https://en.wikipedia.org/wiki/Data_remanence#Overwriting)](https://en.wikipedia.org/wiki/Data_remanence#Overwriting "Data remanence"). Synchronous updates `S` `+S,-S` Changes are written synchronously to the underlying filesystem storage medium; equivalent to 'sync' mount option for affected files.
```
